src/Controller/Admin/AdministrationBaseController.php

```diff
@@ -34,6 +34,13 @@ abstract class AdministrationBaseController extends AppController
      */
     protected $readonly = true;
 
+    /**
+     * Deleteonly flag view.
+     *
+     * @var bool
+     */
+    protected $deleteonly = false;
```

Comment:
Maybe `$allowDelete` or similar? And `'allowDelete'` as string/var below

(sounds like "only delete operation is possible")